engine/Default/sector_move_processing.php

```diff
@@ -88,7 +88,7 @@
 }
 
 // log action
-$player->actionTaken('WalkSector',array('Sector'=>&$sector));
+$account->log(LOG_TYPE_MOVEMENT, 'Walks to sector: ' . $var['target_sector'], $player->getSectorID());
```

Comment:
Can you change this to
```
$player->actionTaken('WalkSector',array('Sector'=>SmrSector::getSector($var['target_sector'])));
```
instead, that way it should still fix the bug, and still allows the ability for missions to distinguish between walking to a sector and getting there via other means

Reply:
I can make this change, but first I just want us to briefly consider if it's necessary. I would like to eventually add a mission for each race (so that Salvene isn't uniquely advantaged), but I can't think of any case where I'd want to penalize someone for jumping into a sector. If anything, it seems more likely to be used accidentally if a coder isn't aware of `EnterSector`.

I leave it up to you, though, so let me know. Thanks!

Reply:
I think it's nice to have, in that it lets the logging be largely in a standardised place and it offers up the power of distinguishing between types of movement in missions, although just because it's there doesn't mean it has to be used (I think there's a fair few "missionable" actions right now that no missions use)

Reply:
Fair enough! I'll make this change shortly.